subprojects/gradle/cd/src/main/kotlin/com/avito/ci/ReleaseBuildStepsExtension.kt

```diff
@@ -0,0 +1,15 @@
+package com.avito.ci
+
+import com.avito.ci.steps.release.DeployStep
+import groovy.lang.Closure
+import org.gradle.api.model.ObjectFactory
+
+
+class ReleaseBuildStepsExtension(
+    objects: ObjectFactory
+) : BuildStepListExtension("release", objects) {
```

Comment:
> deploy step is accessible only from release {}

Why do we need this restriction?

I'd like to remove built-in builds from the Cd plugin (see MBS-7948).

Reply:
Let discuss MBS-7948 and then return here.

Reply:
We decided to leave the abilities of flexible steps configuration.
Because we have a lot of scenarios where our consumers need this flexibility.